flasher_stub/README.md

```diff
@@ -0,0 +1,3 @@
+This is the source of CesantaFlasher.FLASHER_STUB.
+
+Environment to compile it is available [here](https://github.com/cesanta/fnc/tree/master/common/platforms/esp8266/stubs).
```

Comment:
ok, done.
it still depends o docker image, but it shouldn't be a problem to remove that by adding a SDK_PATH variable or some such. i do not actually have the toolchain installed to test w/o docker, so i'll leave it to you. should be trivial.